### PR TITLE
small corrections, clarifications and consistency

### PR DESCRIPTION
**Pull request by juliogomez**

### CODE REVIEW

File: lab.md [hunk 1]
```diff
@@ -1205,7 +1241,6 @@ Each exercise also includes a Python script file that can be executed directly.
     ```python
     command = show_interface_config_temp.format("Loopback103")
     interface = ch.send_command(command)
-    print(interface)
```
Comment (hpreston) [inline lab.md]: I actually want the error to display.  It's part of the talk track during the lab why CLI based interactions aren't as good.  

Please put the `print` statement back in, but adding a note in the step to the effect of 

* *Note: attempting to view the configuration of a non-existing interface will generate a CLI error.  This output is expected, and one of the reasons APIs like NETCONF or RESTCONF are better suited to programmatic interactions.*

**Review by hpreston — CHANGES_REQUESTED** [review]
@juliogomez these are great, thanks for going through.  I made one comment for a change on the netmiko removal of "print".  If you can fix that and update the PR I'll get it all merged in before the class this morning.

File: lab.md (hunk 1 above)
Comment (juliogomez) [reply lab.md]: cool, will do it immediately

**Review by hpreston — APPROVED** [review]
Great, thanks!

**Pull request merged**